src/main/java/ai/chat/dto/UserResponseDto.java

```diff
@@ -0,0 +1,21 @@
+package ai.chat.dto;
+
+import ai.chat.model.AccountType;
+
+public class UserResponseDto {
+    private String id;
+    private AccountType accountType;
```

Comment:
we can add usename field here as well, as we are giving generic info about the user by id without passwords of course.